CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## main / unreleased
+* [BUGFIX] Debian package: Fix post-install, environment file path and user creation. #3720 
```

Comment:
Could you move this to the `### Grafana Mimir` section? That's where packaging entries have gone in the past.

Reply:
Hi, thanks for the review, i moved the comment to the dedicated section and fixed the trailing whitespace.